cocinaborcelle/src/styles/globals.css

```
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body,html {
+  height: 100vh;
+}
```

```suggestion
body,html {
  width: 100vw;
  overflow-x: hidden;
}

html {
  min-height: 100vh;
  position: relative;
}
```

setar 100vh para o body não tá fazendo efeito. Recomendo utilizar esta estilização global

no body, html, estou evitando que algum background possa transbordar da página e fazer aparecer um scroll horizontal, então limito a largura da página e, em caso de overflow, escondo o excesso de conteúdo com a opção hidden

no html, estou utilizando essas propriedades para melhor posicionar o footer. Como estava antes, o footer ficava fixo na tela e por cima do conteúdo da página, mas nós queremos que ele seja jogado para baixo do conteúdo da página e não que fique por cima